types/node/index.d.ts

```diff
@@ -783,7 +791,9 @@ declare namespace NodeJS {
         chdir(directory: string): void;
         cwd(): string;
         debugPort: number;
+        emitWarning(warning: string | Error, options?: WarningOptions): void;
         emitWarning(warning: string | Error, name?: string, ctor?: Function): void;
+        emitWarning(warning: string | Error, name?: string, code?: string, ctor?: Function): void;
```

Comment:
I think `name` and `code` should be mandatory but `string | undefined` here as the single arg version is already covered.